docs/reporting-services/report-data/xml-connection-type-ssrs.md

```diff
@@ -38,7 +38,7 @@ ms.workload: "On Demand"
 |XML document|`http://localhost/XML/Customers.xml`|  
 |Embedded XML document|*Empty*|  
   
- For more connection string examples, see [Data Connections, Data Sources, and Connection Strings in Report Builder](http://msdn.microsoft.com/library/7e103637-4371-43d7-821c-d269c2cc1b34).  
+ For more connection string examples, see [Data Connections, Data Sources, and Connection Strings in Report Builder](../../reporting-services/report-data/data-connections-data-sources-and-connection-strings-report-builder-and-ssrs.md).  

```

Comment:
i see that the change is not a URL, but a pointer to a different file.  I did check the original URL and that is no longer a useful link.  If the new pointer is directing to the samples, then I have no issue with the change.  Was not able to check that myself though.